### PR TITLE
Update assembly.rst

### PR DESCRIPTION
**Pull request by LykleSchepers**
Added the bottom part about running etc. 
Only issue here is that for some reason I loose the text of Step 3
The text is in the raw data, but for some reason does not show up, only the **** show. 
I replaced the text, deleted it and wrote it again, nothing helps

### CODE REVIEW

File: docs/nimble_dual/assembly.rst [hunk 1]
```diff
@@ -1,209 +1,362 @@
...
     :alt: Insert nuts into Dual base
-    :align: center
-    :figclass: align-center
     :height: 400px
-    :width: 400px
+    :width: 286px
+
+    Insert nuts into Dual base
+
+    *Take the Dual Nimble base (Part D) and place it upside down on the table.
```
Comment (BrianGilbert) [inline docs/nimble_dual/assembly.rst]: You need a space after the asterisk for it to diaply as a list.

**Comment by BrianGilbert** [comment]
Looks like a lot of your list items are missing a space after the asterisk, add those in and I'll merge this.

**Review by LykleSchepers — COMMENTED** [review]
ok done

**Comment by LykleSchepers** [comment]
added the spaces and it looks better now

**Pull request merged**